cypress/integration/main_page_spec.js

```diff
@@ -1,25 +1,44 @@
-import { shouldBeRedirectedToLogin } from '../support/assertions';
+import {
+  shouldBeRedirectedToLogin,
+  shouldHaveApiVersionNumber
+} from '../support/assertions';
 
-describe('Dashboard Tests', () => {
+describe('Dashboard Home Page', () => {
   it('When not logged in it should redirect to login page', () => {
     cy.visit('/');
     shouldBeRedirectedToLogin();
   });
 
-  it('Logging in successfully redirects to the Dashboard main page', () => {
-    cy.get('div[class=modal__internal]').within(() => {
-      cy.get('a').click();
+  describe('When logged in', () => {
+    beforeEach(() => {
+      cy.visit('/');
+      cy.get('div[class=modal__internal]').within(() => {
```

Comment:
This is super nitpicky, but in theory, we only need to test the manual login process (actually clicking the login link) once, whereas any other tests depending on logged in state can use the `cy.login` helper. 

So I think I might prefer if we move the "Logging in succesfully..." test back outside of this describe and use this code to login manually in it. Then in this block we can use `cy.login()` in `beforeEach` and leave your compatible API version number test as is.